cmd/controller/main.go

```diff
@@ -60,7 +62,7 @@ func init() {
 
 const (
 	//(cbrookes) This will be removed in the future when we have many tenant ns and way to map to them
-	defaultCtrlNS       = "argocd"
+	defaultCtrlNS       = "multi-cluster-traffic-controller-system"
```

Comment:
@maleck13 Why was this "argocd" up to now, and not the ns that we intend the controller to run in? Just trying to understand what this was for previously.

Reply:
It was because we were using the secrets in the argocd ns

Reply:
Does it still need to be argocd? Do the secrets have to be in that ns?

Reply:
@david-martin Will changing this cause any issues? We shoudl probably just be adding resources (DNSRecord/ManagedZones) into the same NS the controller is running in, but changing this was the quickest way to do it, and works when you are running it locally.

Reply:
No.
I had changed that too on the gateway reconciler branch https://github.com/Kuadrant/multi-cluster-traffic-controller/pull/84/files#diff-369ede0a9a64763f9a3f47aa07fa8a9d1062b3df2783458f02e49cf7f1f3554dR63